shell/platform/embedder/tests/embedder_config_builder.cc

```diff
@@ -132,6 +146,20 @@ void EmbedderConfigBuilder::SetSnapshots() {
   }
 }
 
+void EmbedderConfigBuilder::SetAotDataElf() {
+  FlutterEngineAOTDataSource data_in;
+  FlutterEngineAOTData data_out;
+
+  const auto elf_path =
+      fml::paths::JoinPaths({GetFixturesPath(), kAOTAppELFFileName});
+
+  data_in.type = kFlutterEngineAOTDataSourceTypeElfPath;
+  data_in.elf_path = elf_path.c_str();
+
+  FlutterEngineCreateAOTData(&data_in, &data_out);
```

Comment:
Missing error handling. Return bool from this method for the test to handle or just assert success.